lib/afmotion/http_client.rb

```diff
@@ -62,6 +65,12 @@ def request_serializer(serializer)
       else
         @operation_manager.requestSerializer = serializer
       end
+      reinit_options
+    end
+
+    def reinit_options
```

Comment:
Seems like `reapply` would be a more precise verb. The options objects are already initialized